specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-06-02-preview/fleets.json

```diff
@@ -0,0 +1,741 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/FleetNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/FleetMembersListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Lists the members of a fleet.": {
+            "$ref": "./examples/FleetMembers_List.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Fleet": {
+      "type": "object",
+      "properties": {
+        "etag": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource Etag."
+        },
+        "properties": {
+          "description": "Properties of a fleet.",
+          "$ref": "#/definitions/FleetProperties",
+          "x-ms-client-flatten": true
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "The fleet resource which contains multiple Kubernetes clusters as its members."
+    },
+    "FleetProperties": {
+      "type": "object",
+      "description": "Properties of a fleet.",
+      "properties": {
+        "hubProfile": {
+          "description": "The FleetHubProfile configures the fleet's hub. it is enabled by default and can be disabled by setting the hubMode property.",
+          "$ref": "#/definitions/FleetHubProfile",
+          "x-ms-client-flatten": true
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "x-ms-enum": {
+            "name": "FleetProvisioningState",
+            "modelAsString": true
+          },
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Canceled",
+            "Creating",
+            "Deleting",
+            "Updating"
+          ],
+          "description": "The provisioning state of the last accepted operation."
+        }
+      }
+    },
+    "FleetHubProfile": {
+      "type": "object",
+      "description": "The FleetHubProfile configures the fleet hub. it is enabled by default and can be disabled by setting the hubMode property.",
```

Comment:
Same fix as above.

Reply:
I also don't see a `hubMode` property on this object? Or anywhere in this file acutally?